DE/VR-DE-0003/rule.json

```diff
@@ -33,33 +33,107 @@
   "ValidTo": "2030-06-01T00:00:00Z",
   "AffectedFields": [
     "v.0",
-    "v.0.dt"
+    "v.0.dt",
+    "v.0.mp",
+    "v.0.dn",
+    "v.0.sd"
   ],
   "Logic": {
     "if": [
       {
         "var": "payload.v.0"
       },
       {
-        "not-before": [
-          {
-            "plusTime": [
-              {
-                "var": "external.validationClock"
-              },
-              0,
-              "day"
-            ]
-          },
-          {
-            "plusTime": [
-              {
-                "var": "payload.v.0.dt"
-              },
-              15,
-              "day"
-            ]
-          }
+        "in": [
```

Comment:
So, this is effectively `or: [ ... ]`, right? :)

Reply:
`{ "!": [ { "and": [ { "!": [ <operand_1> ] },  { "!": [ <operand_2> ] }, ...,  { "!": [ <operand_n> ] } ] } ] }` would also work

Reply:
yes, the wish was 'or', so 'in' is reality.
first draft was { "if": [ { arg1 }, true, { "if": [ { arg2 }, true, { "if": [ { arg3 }, true, { "if": ... }]}]}]}